### DISCUSSION TITLE
Catalan translation

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#298] [Q&A] [answered]
@airald thanks for the impressive work! :+1: 

Catalan translation and voice navigation will be available in next Cruiser.

**Comment by ramondevic** [comment]
Oh, that's great!

Merci @airald

**Reply by airald** [reply]
You can contact me to discuss it if you find mistakes or something that can be enhanced. I'm sure that some of them are also in my list.

**Comment by devemux86** [comment, marked as answer]
Catalan translation and voice navigation are available in [Cruiser 3.0.32](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

@airald thanks for your help! :+1:

**Comment by ramondevic** [comment]
There is a problem with the Catalan translation in "Map Layers > Camping ..."

<img width="300" height="650" alt="Screenshot_20260609-165229" src="https://github.com/user-attachments/assets/06e8e5fe-e40d-4fc2-81af-75a58a59c470" />
<img width="300" height="650" alt="Screenshot_20260609-165132" src="https://github.com/user-attachments/assets/1305565a-3745-4eee-8ffd-937940f8bcfb" />

**Reply by devemux86** [reply]
@ramondevic Thanks for the report, it will be fixed in the next update.
